bochs/config.h.in

```diff
@@ -808,8 +808,8 @@ typedef Bit32u bx_phy_address;
   #error To enable USB, you must also enable PCI
 #endif
 
-#if (BX_SUPPORT_USB_EHCI && !BX_SUPPORT_USB_UHCI)
-  #error To enable EHCI, you must also enable UHCI
+#if (BX_SUPPORT_USB_EHCI && !BX_SUPPORT_USB_UHCI && !BX_SUPPORT_USB_OHCI)
```

Comment:
I would duplicate the check in the configure script too

Reply:
> I would duplicate the check in the configure script too

Do you mean in `configure` and `configure.ac`? I am completely unaware of how those scripts are used.
I did not see anything about this check in either file. Is the original check for UHCI contained in them or is it missing as well?

Does the whole check need to be added to each file? If so, I am afraid it will have to be someone else, since I do not understand the script files, nor how to write/modify them correctly.

Thanks,
Ben

Reply:
Thank you. It is appreciated.